components/engine_traits/src/engine.rs

```diff
@@ -162,14 +162,16 @@ impl Drop for TabletErrorCollector {
 }
 
 /// OpenOptionsn is used for specifiying the way of opening a tablet.
-#[derive(Default, Clone)]
+#[derive(Default, Clone, Debug)]
 pub struct OpenOptions {
     // create tablet if non-exist
     create: bool,
     create_new: bool,
     read_only: bool,
     cache_only: bool,
     skip_cache: bool,
+    split_use: bool,
```

Comment:
This is too strange. We have `skip_cache` and `open_tablet_raw`, the combination is for the exact purpose.

Reply:
Ok，done.